include/spdlog/details/logger_impl.h

```diff
@@ -87,6 +88,9 @@ inline void spdlog::logger::log(level::level_enum lvl, const char* msg)
     {
         details::log_msg log_msg(&_name, lvl);
         log_msg.raw << msg;
+        if (_formatter->do_msg_count()) {
```

Comment:
This is nice.. (though I wonder if the cost of calling ```do_msg_count()``` is similar or to the atomic increment  though..)